### PR TITLE
Add descriptions for queries R-W

### PR DESCRIPTION
**Pull request by abdonpijpelink**
Relates to https://github.com/elastic/elasticsearch-specification/issues/964

Adds Query DSL descriptions for queries R through W. Descriptions were sourced from https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html and subpages.

### CODE REVIEW

File: specification/_types/query_dsl/span.ts [hunk 1]
```diff
@@ -42,27 +56,57 @@ export class SpanFirstQuery extends QueryBase {
...
+   * The number of tokens from within the include span that can’t have overlap with the exclude span.
+   * Equivalent to setting both `pre` and `post`.
+   */
   dist?: integer
+  /**
+   * Span query whose matches must not overlap those returned.
+   */
   exclude: SpanQuery
+  /**
+   * Span query whose matches are filtered
```
Comment (szabosteve) [inline specification/_types/query_dsl/span.ts:91]:
```suggestion
   * Span query whose matches are filtered.
```

File: specification/_types/query_dsl/span.ts [hunk 2]
```diff
@@ -42,27 +56,57 @@ export class SpanFirstQuery extends QueryBase {
...
+  /**
+   * The number of tokens before the include span that can’t have overlap with the exclude span.
+   * @server_default 0
+   */
   pre?: integer
 }
 
 export class SpanOrQuery extends QueryBase {
+  /**
+   * Array of one or more other span type queries
```
Comment (szabosteve) [inline specification/_types/query_dsl/span.ts:108]:
```suggestion
   * Array of one or more other span type queries.
```

File: specification/_types/query_dsl/term.ts [hunk 3]
```diff
@@ -198,7 +273,10 @@ export class WildcardQuery extends QueryBase {
    * @availability serverless
    */
   case_insensitive?: boolean
-  /** Method used to rewrite the query */
+  /**
+   * Method used to rewrite the query
```
Comment (szabosteve) [inline specification/_types/query_dsl/term.ts:277]:
```suggestion
   * Method used to rewrite the query.
```

**Review by szabosteve — APPROVED** [review]
LGTM. I left the tiniest possible suggestion - three times. Please take them or leave them.

**Comment by abdonpijpelink** [comment]
Thanks @szabosteve !

**Pull request merged**